components/provider_ppucd/src/error.rs

```diff
@@ -0,0 +1,34 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+use std::fmt;
+
+#[derive(Debug)]
+pub enum Error {
+    PpucdParse(PpucdParseError),
+}
+
+#[derive(Debug, PartialEq, Copy, Clone)]
+pub struct PpucdParseError {
+    pub src: &'static str,
```

Comment:
That's a bit odd. `provider_cldr` uses `PathBuf` here. Why `&'static str`?

Reply:
Although the parsing code started off taking a file as an input, it currently takes a `&str` of the contents of a PPUCD file, not the `File` itself. But it probably should add an function that can accept a File when integrating with the exporter tool. I'm not sure what the pros and cons `str` vs `File` would be at that point, though.

Reply:
Why is it `'static` though?